### PR TITLE
Add new configuration for Modbus Controller.

### PR DESCRIPTION
**Pull request by 0x3333**
## Description:

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#5318.

## Checklist:

  - [x] I am merging into `next` because this is new documentation that has a matching pull-request in [esphome](https://github.com/esphome/esphome) as linked above.  
    or
  - [ ] I am merging into `current` because this is a fix, change and/or adjustment in the current documentation and is not for a new component or feature.

  - [ ] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

**Review by nagyrobi — COMMENTED** [review]
Is this a true/false option?

**Comment by jesserockz** [comment]
As this is a feature matched with a PR in `esphome/esphome`, please target your PR to the `next` branch and rebase.

Thanks
Jesse

File: components/modbus_controller.rst [hunk 1]
```diff
@@ -59,6 +59,11 @@ Configuration variables:
 - **update_interval** (*Optional*, :ref:`config-time`): The interval that the sensors should be checked.
   Defaults to 60 seconds.
 
+- **offline_skip_updates** (*Optional*, :ref:`offline_skip_updates`): When a controller doesn't respond to a command, it is
```
Comment (jesserockz) [inline components/modbus_controller.rst]:
```suggestion
- **offline_skip_updates** (*Optional*, integer): When a controller doesn't respond to a command, it is
```

**Comment by 0x3333** [comment]
> Is this a true/false option?

Sorry, my bad, its an `integer`.

**Comment by 0x3333** [comment]
> As this is a feature matched with a PR in `esphome/esphome`, please target your PR to the `next` branch and rebase.

Done.

**Review by jesserockz — APPROVED** [review]

**Pull request merged**